src/overview/comparing-std-and-no_std.md

```diff
@@ -1,69 +1,37 @@
-# Comparing `std` and `no_std`
+# Standard library (`std`) vs. Bare Metal (`no_std`)
 
-There are several factors that must be considered when choosing between `std` ([esp-idf-hal]) and `no_std` (eg. [esp-hal]). As stated previously, each approach has its own unique set of advantages and disadvantages. While we can't decide for you, this section will hopefully allow you to make an educated decision.
+## Support for Espressif Products
 
-[esp-idf-hal]: https://github.com/esp-rs/esp-idf-hal
-[esp-hal]: https://github.com/esp-rs/esp-hal
+|   Chip   | `std` | `no_std` |
+| -------- | :--------: | :---: |
+|  ESP32   |     ✅      |     ✅      |
+| ESP32-C2 | _planned_   |     ✅      |
+| ESP32-C3 |     ✅      |     ✅      |
+| ESP32-C6 |      ?      | _planned_  |
+| ESP32-S2 |     ✅      |     ✅      |
+| ESP32-S3 |     ✅      |     ✅      |
+| ESP32-H2 | _planned_   | _planned_  |
+| ESP8266  |     ❌      |     ✅      |
 
-## Application Runtimes
+The products supported in certain circumstances will be called _supported Espressif products_ throughout the book.
 
-In the case of applications (as opposed to libraries) the standard library provides a runtime that handles setting up stack overflow protection, spawning the main thread before an application's `main` function is invoked, and handling of command-line arguments.
+As of now, the Espressif products supported by the esp-idf framework are the ones supported for Rust `std` development.
```

Comment:
This should be explained in the `std` chapter and this is not something temporal.

Reply:
Copied this paragraph to the `std` chapter and added the link to the esp-idf releases and SoC compatibility table.